Errors compiling on manjaro

[darkstar@WOPR gorc]$ make 
[  3%] Built target utility
[  5%] Built target log
[  7%] Built target io
[  9%] Built target text
[ 10%] Built target build-common
[ 11%] Built target system
[ 14%] Built target program
[ 15%] Built target boc
[ 15%] Built target ast
[ 18%] Built target boc-shell
[ 19%] Built target boc-test
[ 20%] Built target vfs
[ 22%] Built target content
[ 25%] Built target cog-script
[ 26%] Built target cog-ast
[ 29%] Built target cog-vm
[ 30%] Built target cog-ir
[ 31%] Built target cog-codegen
[ 31%] Built target cog-grammar
[ 34%] Built target cog-semantics
[ 35%] Built target cog-compiler
[ 37%] Built target jk-content
[ 37%] Building CXX object src/jk/vfs/CMakeFiles/jk-vfs.dir/episode_entry_type.cpp.o
/home/darkstar/gorc/src/jk/vfs/episode_entry_type.cpp: En la función ‘const char* gorc::as_string(gorc::episode_entry_type)’:
/home/darkstar/gorc/src/jk/vfs/episode_entry_type.cpp:22:16: error: ‘range_error’ no es un miembro de ‘std’
   22 |     throw std::range_error("not a valid episode_entry_type");
      |                ^~~~~~~~~~~
/home/darkstar/gorc/src/jk/vfs/episode_entry_type.cpp:23:1: aviso: el control alcanza el final de una función que no es void [-Wreturn-type]
   23 | }
      | ^
make[2]: *** [src/jk/vfs/CMakeFiles/jk-vfs.dir/build.make:109: src/jk/vfs/CMakeFiles/jk-vfs.dir/episode_entry_type.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:2243: src/jk/vfs/CMakeFiles/jk-vfs.dir/all] Error 2
make: *** [Makefile:104: all] Error 2
[darkstar@WOPR gorc]$

--- a/src/jk/vfs/episode_entry_type.cpp
+++ b/src/jk/vfs/episode_entry_type.cpp
@@ -1,6 +1,7 @@
 #include "episode_entry_type.hpp"
 #include "utility/enum_hash.hpp"
 #include <unordered_map>
+#include <stdexcept>
 
 using namespace gorc;
 

